src/runnerComponent.js

```diff
@@ -132,7 +132,7 @@ export default function runnerComponentFactory(container = null, config = {}, te
         })
         .on('render', function() {
             const runnerConfig = Object.assign(_.omit(this.config, ['providers']), {
-                renderTo: this.getElement()
+                renderTo: this.getElement().get(0)
```

Comment:
it looks like there are still some external expectation for it to be a jquery element :/